frontend/packages/console-app/src/components/access-modes/access-mode.tsx

```diff
@@ -33,6 +33,7 @@ export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) =>
     availableAccessModes = [],
     description,
     ignoreReadOnly,
+    accMode,
```

Comment:
nitpick: Could we name this something like `defaultAccessMode` or `initialAccessMode`?

Reply:
+1 changed to `initialAccessMode`